docs/Configuration.md

```diff
@@ -470,6 +474,175 @@ Example configuration with example log can be found in [/examples/logs_json/](/e
 [filelogreceiver_readme]: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/v0.47.0/receiver/filelogreceiver
 [opentelemetry-log-collection]: https://github.com/open-telemetry/opentelemetry-log-collection
 
+## Setting source category
+
+For many Sumo Logic customers the [source category][source_category_docs] is a crucial piece of metadata
+that allows to differentiate data coming from different sources.
+This section describes how to configure the Sumo Logic Distribution of OpenTelemetry
+to decorate data with this attribute.
+
+To decorate all the data from the collector with the same source category,
+set the `source_category` property on the [Sumo Logic exporter][sumologicexporter_docs] like this:
+
+```yaml
+exporters:
+  sumologic:
+    source_category: my/source/category
+```
+
+You can also use Sumo Logic exporter's [source templates][sumologicexporter_source_templates]
+to use other attributes' values to compose source category, like this:
+
+```yaml
+exporters:
+  sumologic:
+    source_category: "%{cloud.provider}/%{cloud.region}/%{cloud.platform}"
+```
+
+If you want data from different sources to have different source categories,
+you'll need to set a resource attribute named `_sourceCategory` earlier in the pipeline.
```

Comment:
```suggestion
you'll need to set a resource attribute named (for example) `_sourceCategory` earlier in the pipeline.
```

Reply:
The attribute needs to specifically be named `_sourceCategory` to be used in Sumo as the source category.

Reply:
Do we have that documented somewhere in https://help.sumologic.com/ ?